src/ucp/wireup/wireup.c

```diff
@@ -30,6 +30,48 @@ static size_t ucp_wireup_msg_pack(void *dest, void *arg)
     return sizeof(ucp_wireup_msg_t) + req->send.length;
 }
 
+static const char* ucp_wireup_msg_str(uint8_t msg_type)
+{
+    switch (msg_type) {
+    case UCP_WIREUP_MSG_PRE_REQUEST:
+        return "PRE_REQ";
+    case UCP_WIREUP_MSG_REQUEST:
+        return "REQ";
+    case UCP_WIREUP_MSG_REPLY:
+        return "REP";
+    case UCP_WIREUP_MSG_ACK:
+        return "ACK";
+    default:
+        return "<unknown>";
+    }
+}
+
+static ucp_lane_index_t ucp_wireup_get_msg_lane(ucp_ep_h ep, uint8_t msg_type)
+{
+    const char      *unified_mode_msg = ". try to set UCX_UNIFIED_MODE=n.";
+    ucp_context_h   context           = ep->worker->context;
+    ucp_ep_config_t *ep_config        = ucp_ep_config(ep);
+    ucp_lane_index_t lane             = UCP_NULL_LANE;
+
+    if (msg_type != UCP_WIREUP_MSG_ACK) {
+        /* for request/response, try wireup_lane first */
+        lane = ep_config->key.wireup_lane;
+    }
+
+    if (lane == UCP_NULL_LANE) {
+        /* fallback to active messages lane */
+        lane = ep_config->key.am_lane;
+    }
+
+    if (lane == UCP_NULL_LANE) {
+        ucs_fatal("ep %p to %s: could not fine a lane to send CONN_%s%s",
+                  ep, ucp_ep_peer_name(ep), ucp_wireup_msg_str(msg_type),
+                  context->config.ext.unified_mode ? unified_mode_msg : "");
```

Comment:
can we use ". try to set UCX_UNIFIED_MODE=n." directly instead of setting it to `unified_mode_msg`?
i.e. (it will be shorten than 80 symbols per line)
```
ucs_fatal("ep %p to %s: could not fine a lane to send CONN_%s%s",
          ep, ucp_ep_peer_name(ep), ucp_wireup_msg_str(msg_type),
          context->config.ext.unified_mode ?
          ". try to set UCX_UNIFIED_MODE=n." : ".");
```
or
```
ucs_fatal("ep %p to %s: could not fine a lane to send %s%s",
          ep, ucp_ep_peer_name(ep), ucp_wireup_msg_str[msg_type)],
          context->config.ext.unified_mode ?
          ". try to set UCX_UNIFIED_MODE=n." : ".");
```